jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -223,12 +225,22 @@ public Span log(long timestampMicroseconds, Map<String, ?> fields) {
         if (logs == null) {
           this.logs = new ArrayList<LogData>();
         }
+
+        if (tracer.expandExceptionLogs()) {
+          Map<String, Object> errorFields = exceptionLogs(fields);
+          if (!errorFields.isEmpty()) {
+            errorFields.putAll(fields);
+          }
+          fields = errorFields;
```

Comment:
it looks like if the fields don't contain "error.object", we throw out all the fields the user passed in?